src/less/tooltip/base/tooltip.less

```diff
@@ -162,6 +162,66 @@ span.tourtip {
     width: @icon-width;
 }
 
+.dialog-infotip[role="dialog"] {
+    .dialog-base();
+}
+
+.dialog-infotip__window {
+    .dialog-window();
+
+    padding-bottom: 15vh;
+    top: 15%;
+    width: calc(100% - 32px);
+
+    &::before {
+        border-radius: 7px;
+    }
+}
+
+.dialog-infotip__window:not(.dialog-infotip__window--full) {
+    flex-direction: row-reverse;
+    left: 50%;
+    max-width: 400px;
+    min-width: 200px;
+    top: 50%;
+    transform: translate(-50%, -25%);
```

Comment:
To center the dialog horizontal and vertical

Reply:
So that's fine, I'm just wondering why it's `-25%` and not `-50%` to center it 
https://css-tricks.com/centering-css-complete-guide/ (In the centering both vertically and horizontally and unknown width/height)

Reply:
I used that same guideline but `-50%` didn't exactly center it :)